Resources/views/Base/tree.html.twig

```diff
@@ -1,10 +1,10 @@
 <script type="text/javascript" src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
 <script type="text/javascript" src="{{ path('fos_js_routing_js', {"callback": "fos.Router.setData"}) }}"></script>
 
-<script src="{{ asset('bundles/cmftreebrowser/js/jquery.jstree.js') }}" type="text/javascript"></script>
-<script src="{{ asset('bundles/cmftreebrowser/js/jquery.cookie.js') }}" type="text/javascript"></script>
+<script>window.jQuery || document.write(unescape('%3Cscript src="{{ asset('bundles/cmf/treebrowser/js/vendor/jquery/dist/jquery.min.js') }}"%3E%3C/script%3E'))</script>
+<script>jQuery.ui || document.write(unescape('%3Cscript src="{{ asset('bundles/cmf/treebrowser/js/vendor/jquery-ui/jquery-ui.min.js') }}"%3E%3C/script%3E'))</script>
+<script src="{{ asset('bundles/cmftreebrowser/js/vendor/fancytree/dist/jquery.fancytree-all.min.js') }}" type="text/javascript"></script>
+<script src="{{ asset('bundles/cmftreebrowser/js/adapter/fancytree.js') }}"></script>
+<script src="{{ asset('bundles/cmftreebrowser/js/jquery.cmf_tree.js') }}"></script>
```

Comment:
imho we have to start using assetic here, to combine all the js files into one. we do that in the CreateBundle for example. 

people who think assetic is not cool enough can run their own stuff to combine files and override this template